doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,7 +202,13 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added handling of :c:struct:`hid_report_subscriber_event` to :ref:`nrf_desktop_hid_forward`.
```

Comment:
Remove the changelog entry from this PR and add it as a comment to Divya's https://github.com/nrfconnect/sdk-nrf/pull/18192.

Reply:
Added comment. As soon as it's applied, I will remove release notes from this PR.